### PR TITLE
Add CODEOWNERS file

### PR DESCRIPTION
**Pull request by jbech-linaro**
First step in automate review requests. @OP-TEE/linaro works as a catch-all and the rest is more or less for platform specific things.

// Joakim

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# This file should stay in somewhat sync with the MAINTAINERS file, since there
+# is some kind of relation between the two. Preferably use GitHub alias here
+# instead of email addresses.
+
+# Everything not uniquely specified should land at the core team
+* @OP-TEE/linaro
+
+# Platform specific code
+core/arch/arm/plat-d02/ @OP-TEE/plat-d02
```
Comment (jbech-linaro) [inline .github/CODEOWNERS]: Start all root files with `/` (according to the [example](https://help.github.com/articles/about-codeowners/#codeowners-file-example)).

**Comment by jbech-linaro** [comment]
There is some duplication with the maintainer file, but it fulfills another purpose also and that is automate things for us. There is an initial cost for setting this up (it's what I've been doing now), but after that the maintenance for this is low and it will help me keep track of things for the project and what is going on at our GitHub pages.

**Comment by jbech-linaro** [comment]
I need it. Let's discuss further in our 1:1 about who is doing what.

**Comment by jforissier** [comment]
You need it for what? This is a public project, please share your arguments publicly. What is wrong with our current work flow that needs fixing and that this PR would help fixing?

**Comment by jbech-linaro** [comment]
> You need it for what? This is a public project, please share your arguments publicly. What is wrong with our current work flow that needs fixing and that this PR would help fixing?

I've sent an email to all maintainers earlier today explaining why I need it. I will update the commit message to make it clearer.

**Comment by jbech-linaro** [comment]
Commit message updated (I had to force push to get the updated commit message visible on the first patch).

**Comment by jbech-linaro** [comment]
As long as we need to grant write access to the actual team in question, we cannot use this feature: Therefore I'm closing this now and will eventually re-open it in the future it the write access requirement disappears.

**Pull request closed without merging**